data/slack-jackson-dto/src/main/kotlin/io/olaph/slack/dto/jackson/group/users/SetPresence.kt

```diff
@@ -0,0 +1,31 @@
+package io.olaph.slack.dto.jackson.group.users
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.annotation.JsonSubTypes
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import io.olaph.slack.dto.jackson.JacksonDataClass
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "ok", visible = true)
+@JsonSubTypes(
+        JsonSubTypes.Type(value = SuccessfulUsersSetPresenceResponse::class, name = "true"),
+        JsonSubTypes.Type(value = ErrorUsersSetPresenceResponse::class, name = "false")
+)
+@JacksonDataClass
+sealed class UsersSetPresenceResponse constructor(@JsonProperty("ok") open val ok: Boolean)
+
+@JacksonDataClass
+data class SuccessfulUsersSetPresenceResponse constructor(override val ok: Boolean)
+    : UsersSetPresenceResponse(ok) {
+    companion object
+}
+
+@JacksonDataClass
+data class ErrorUsersSetPresenceResponse constructor(override val ok: Boolean,
+                                                     @JsonProperty("error") val error: String)
+    : UsersSetPresenceResponse(ok) {
+    companion object
+}
+
+data class SlackUsersSetPresenceRequest(@JsonProperty("presence") val presence: String) {
```

Comment:
since only "auto" or "away" are possible, i would suggest creating a enum class for that